bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -763,7 +832,16 @@ impl Vm {
             inner: InnerAsyncResult(ctx),
         })
     }
-
+    pub async fn run_registered_function_async2(
+        &self,
+        mod_name: impl AsRef<str> + Send,
+        func_name: impl AsRef<str> + Send,
+        params: impl IntoIterator<Item = WasmValue> + Send,
+    ) -> WasmEdgeResult<Vec<WasmValue>> {
+        FiberFuture::on_fiber(|| self.run_registered_function(mod_name, func_name, params))
+            .await
+            .unwrap()
+    }
```

Comment:
Please replace the original `run_registered_function_async` with the new `run_registered_function_async2`, but keep using the original API name `run_registered_function_async`. Thanks!